app/api/wallet/transactions.go

```diff
@@ -103,7 +103,7 @@ func getRawTransactions() http.Handler {
 			return
 		}
 
-		apiResp.Data = resp
+		apiResp.Data = resp.Results
```

Comment:
I feel this is in the wrong spot? Is there somewhere further up the chain that puts the data into response.Results in the first place?

Could we change it there to use response.Data at that top level?

Reply:
I'm not to sure what you mean, but this is the highest it can possibly be in the stack as this is the API handler function?